.gitignore

```diff
@@ -15,3 +15,4 @@ Cargo.lock
 
 # Generated by dogfood
 /target_recur/
+.*.sw*
```

Comment:
you can set a user-wide gitignore, then those files are ignored in all git repositories of your user. At least you can do that on linux/mac.